runHarness/ComputeResources/KubernetesCluster.pm

```diff
@@ -517,7 +691,45 @@ sub kubernetesGetLogs {
 
 	}
 	
-	return 0;
+	return 1;
```

Comment:
The return value changed, but nobody checks it.
Does it even need a return value? (and for kubernetesCopyFromFirst)